src/app/store.ts

```diff
@@ -97,12 +107,29 @@ export function setUpStore(preloadedState?: Partial<RootState>) {
       maxAge: 1000,
     },
     middleware: (getDefaultMiddleware) => {
+      const production =
```

Comment:
```suggestion
        const production = import.meta.env.PROD;
        
```

Reply:
woops, I just double checked, it should be `import.meta.env.MODE === "production"`